tests/functional/relation_tests/models.py

```diff
@@ -31,3 +31,37 @@
 ) }}
 select * from {{ ref('my_seed') }}
 """
+
+
+DYNAMIC_ICEBERG_TABLE = """
```

Comment:
This technically shouldn't exist yet, but would come in when we merge the iceberg DT PR, and/or when we un-comment line 46 below.

Reply:
I just copied the whole file via your commit to avoid merge conflicts 🫡